src/excel_adapter/excel_adapter.go

```diff
@@ -68,9 +72,31 @@ func fetchDocument(downloadConfig internals.DownloadConfig, parseConfig internal
 		trace.Fatal().Stack().Err(err).Msg("")
 	}
 
+	if !ade_linter.Lint(file) {
+		if !promptContinue() {
+			os.Exit(1)
+		}
+	}
+
 	return internals.GetDocument(file, parseConfig)
 }
 
+func promptContinue() bool {
+	fmt.Println("Continue with incorrect ADE? [Y/n]")
+	for {
+		var selection string
+		fmt.Scanln(&selection)
+
+		if strings.ToUpper(selection) == "Y" || strings.ToUpper(selection) == "YES" {
+			return true
+		} else if strings.ToUpper(selection) == "N" || strings.ToUpper(selection) == "NO" {
+			return false
+		} else {
+			fmt.Println("Incorrect input")
+		}
```

Comment:
add a hint that you can input something again, like `> ` or some thing like that